test/unit/utils.test.ts

```diff
@@ -0,0 +1,9 @@
+import * as assert from 'assert';
+import { normalize_path } from '../../src/utils';
+
+describe('normalize_path', () => {
+	it('lowercases the first letter', () => {
```

Comment:
Awesome!

I think the `assert` signature is `assert.equal(actual, expected)` though, this is backwards.

Reply:
Ah, thanks. That's my Java/JUnit muscle memory. Updated!

Reply:
The way it is now looks like the proper actual, expected ordering to me. However, the strings themselves in the tests need to have the backslashes escaped.

Reply:
good catch! updated